template/interpolate/funcs.go

```diff
@@ -164,6 +165,13 @@ func funcGenTemplateDir(ctx *Context) interface{} {
 	}
 }
 
+// Another idea: create cli function to be used when variable it's inside file passed by cli
```

Comment:
This func cli won't remove the need for a double interpolation. Also, it has to be improved so it could accept other variables besides the ones with format `{{ .%s }}`